services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -361,6 +361,73 @@ class GitLabProjectVersioningTest extends AsyncWordSpec with Matchers with Mocki
       }
     }
 
+    "getFiles" should {
+      val dummyFilesTree: List[FileTree] = List(dummyFileTree)
+      val path = Paths.get(dummyFilesTree.head.path)
+      val encodedPathStr = URLEncoderUtils.encode(path.toString)
+      //        val emptyListValidateResponse = "List((,List(JsonValidationError(List(error.expected.jsarray),WrappedArray())))"
+      val version = dummyPipelineVersion
+
+      def request(project: Project, version: Option[PipelineVersion]): Future[Response[List[FileTree]]] = {
```

Comment:
Let's give it a better name. Something like `getFileTrees` should be good enough

Reply:
I renamed all requests with more understandable names.